27-cellular-connected-electronic-kiosk/README.md

```diff
@@ -0,0 +1,150 @@
+
+![Pi Hat on Pi's headers](images/pi_hat.jpg)
+
+Everything should fit, even with the standoffs attached to the Pi. You will also need to use the micro USB cable to connect the Notecarrier's micro USB port to one of the Pi's USB ports:
+
+![Notecarrier USB connection](images/notecarrier_usb_connection.jpg)
+
+At this point, the assembly should look something like this:
+
+![Fully assembled](images/full_assembly.jpg)
+
+From here, you can continue with the rest of the steps in the Smarti Pi documentation.
+
+### Keyboard and Mouse
+
+Install the AAA batteries into the keyboard and mouse, and plug the USB dongle into a free USB port on the Pi. Note that this single dongle is used for both the keyboard and mouse.
+
+## Booting for the First Time
+
+Plug the USB-C power supply into the exposed USB-C port on the case and let the Pi boot up. Complete the setup steps as described in [this documentation](https://projects.raspberrypi.org/en/projects/raspberry-pi-setting-up/4).
+
+## Notecard Firmware
+
+The Notecard should use [firmware version 5.1.1](https://dev.blues.io/notecard/notecard-firmware-updates/#v5-1-1-april-5th-2023) or higher. The simplest way to update firmware is to do an [over-the-air update](https://dev.blues.io/notecard/notecard-firmware-updates/#ota-dfu-with-notehub).
+
+## Kiosk Software
+
+### Dependencies
+
+To run the kiosk software, you'll need at least Python 3.8. The default Python version that comes with the OS should be sufficient, but you can check the version at the command line with `python --version`. You will also need to install a couple of Python packages. From this directory, `27-cellular-connected-electronic-kiosk`, run `pip install -r requirements.txt` to install them.
+
+### Running
+
+`kiosk.py` is a Python script that runs the application. With `python kiosk.py --help`, you'll see this help message:
+
+```
+usage: kiosk.py [-h] [--data-dir DATA_DIR] --product PRODUCT --route ROUTE
+
+Run the Notecard-based kiosk app.
+
+optional arguments:
+  -h, --help           show this help message and exit
+  --data-dir DATA_DIR  Root directory for downloaded content and other files created by this script.
+  --product PRODUCT    ProductUID for the Notehub project.
+  --route ROUTE        Alias for a Proxy Route in Notehub that will be used to download content.
+```
+
+- `--data-dir` is optional; if you don't specify it, the default data directory will be `~/kiosk-data/`.
+- `--product` should be your [Notehub project's ProductUID](https://dev.blues.io/notehub/notehub-walkthrough/#finding-a-productuid).
+- `--route` should be the `alias` of the proxy route you created.
+
+After launching the script with `python kiosk.py --data-dir <path to your data directory> --product <ProductUID> --route <route alias>`, it will begin downloading the .zip file via the proxy route. Once downloaded, it will unzip the file and open `resources/index.htm` in a browser window.
```

Comment:
I'm debating whether this `README.md` should stay in this folder or be moved over to the other repo [here](https://github.com/blues/accelerators-cellular-connected-electronic-kiosk). 

I think move it, but include a very barebones readme saying something along the lines of "In an effort to keep this accelerator project as small as possible as it's designed to be downloaded via cellular to a Raspberry Pi, please see [this repo](https://github.com/blues/accelerators-cellular-connected-electronic-kiosk) for full details on the Cellular Connected Electronic Kiosk"